src/sequelize/config/config.js

```diff
@@ -5,13 +5,31 @@ module.exports = {
   development: {
     url: process.env.DB_CONNECTION,
     dialect: "postgres",
+    dialectOptions: {
+      ssl: {
+        require: true,
+        rejectUnauthorized: false,
+      },
+    },
```

Comment:
configure this to support localhost because here you are forcing people to use only remote database, you can create env file to handle that

Reply:
this line of code i have added doen't enforce u to use rmot db instead it allow u to connct to remote db either u can connect with local or remote db with these changes